### DISCUSSION TITLE
Sales analytics on the admin dashboard

### DISCUSSION THREAD

**Discussion by lautsevich** [lovata/oc-shopaholic-plugin#363] [Ideas]
We've started working on a basic sales analytics functionality (see PR attached). But we understand there should be a more complex functionality.

Let's help us to make a list of the required functionality. At first glance, we should check Shopaholic competitors on the market, for example, [WooCommerce Admin](https://wordpress.org/plugins/woocommerce-admin/) plugin or something like that.

Please, share our thoughts with us.

**Comment by lautsevich** [comment]
New [home screen](https://woocommerce.wordpress.com/2020/06/26/meet-the-new-woocommerce-home/) in the upcoming minir release of WooCommerce.

![](https://woocommerce.files.wordpress.com/2020/06/home-screen.png?w=1024)

**Comment by lautsevich** [comment]
@Kation-a, please, find the list of possible dashboard UI blocks:

- [List](https://octobercms.com/docs/ui/list)
- [Chart](https://octobercms.com/docs/ui/chart)
- [Scoreboard](https://octobercms.com/docs/ui/scoreboard)

**Comment by wobqqq** [comment]
![download](https://user-images.githubusercontent.com/31685258/102867259-92bb9380-4449-11eb-805f-a1d773236636.png)
![download (1)](https://user-images.githubusercontent.com/31685258/102867260-94855700-4449-11eb-8220-02767600df33.png)
![download (2)](https://user-images.githubusercontent.com/31685258/102867262-951ded80-4449-11eb-8d96-86ffe5e5e0bc.png)
